monokai-theme.el

```diff
@@ -5750,6 +5750,10 @@ Also affects 'linum-mode' background."
      ((,monokai-class (:foreground ,monokai-green))
       (,monokai-256-class (:foreground ,monokai-256-green))))
 
+   `(web-mode-html-tag-bracket-face
+     ((,monokai-class (:foreground ,monokai-orange))
+      (,monokai-256-class (:foreground ,monokai-256-orange))))
```

Comment:
I suggest these should be `gray` not `orange`.

Reply:
Sure, changed.